packages/xo-server/src/xapi/mixins/vm.js

```diff
@@ -257,6 +257,12 @@ export default {
       },
     },
 
+    bootPolicy: {
+      set (bootPolicy, vm) {
+        return this._set('HVM_boot_policy', bootPolicy)
+      },
+    },
```

Comment:
I think we should make the `virtualizationMode` field editable instead (which would in turn update the `HVM_boot_policy` field).

Reply:
I updated this part, is that what you wanted?

Reply:
is that okay now ?

Reply:
Yep, maybe throw if the requested mode is neither `pv` nor `hvm`.

Reply:
fix.